site/content/en/latest/concepts/concepts_overview.md

```diff
@@ -0,0 +1,30 @@
+# Envoy Gateway Resources
+
+There are several resources that play a part in enabling you to meet your Kubernetes ingress traffic handling needs. This page provides a brief overview of the resources you’ll be working with.
+
+# Overview
+
+## Kubernetes Gateway API Resources
+- **GatewayClass:** Defines a class of Gateways with common configuration.
+- **Gateway:** Specifies how traffic can enter the cluster.
+- **Routes:** **HTTPRoute, GRPCRoute, TLSRoute, TCPRoute, UDPRoute:** Define routing rules for different types of traffic.
+## Envoy Gateway (EG) API Resources
+- **EnvoyProxy:** Represents the deployment and configuration of the Envoy proxy within a Kubernetes cluster, managing its lifecycle and settings.
+- **EnvoyPatchPolicy, ClientTrafficPolicy, SecurityPolicy, BackendTrafficPolicy, EnvoyExtensionPolicy, BackendTLSPolicy:** Additional policies and configurations specific to Envoy Gateway.
+
+| API         | Resource                                       | Required | Purpose            | References           | Description                                                                                                                                                                                                 |
+| ----------- | ---------------------------------------------- | -------- | ------------------ | -------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Gateway API | GatewayClass                                   | Yes      | Gateway Config     | Core                 | Defines a class of Gateways with common configuration.                                                                                                                                                      |
```

Comment:
non blocking comment: would be great to add hyperlinks to the resources in a follow up PR, Gateway API links blocked on https://github.com/envoyproxy/gateway/issues/2749

Reply:
Yeah I agree that would be good